### PR TITLE
docs: Added MemGPT integration

### PR DESCRIPTION
**Pull request by Anush008**
## [Preview](https://deploy-preview-637--condescending-goldwasser-91acf0.netlify.app/documentation/frameworks/memgpt/)

Waiting on https://github.com/cpacker/MemGPT/pull/1023 to be released.

### CODE REVIEW

File: qdrant-landing/content/documentation/frameworks/memgpt.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+[MemGPT](https://memgpt.ai/) is a system that enables LLMs to manage their own memory and overcome limited context windows to
+
+- Create perpetual chatbots that learn about you and change their own personalities over time.
+- Create perpetual chatbots that can interface with large data stores.
+
+Qdrant is available as a storage backend in MemGPT for storing and semantically retrieving data.
+
+## Usage
+
+You can configure MemGPT to use either a Qdrant server or an in-memory instance via the `memgpt configure` command.
```
Comment (mjang) [inline qdrant-landing/content/documentation/frameworks/memgpt.md:17]: The Google developer style guide doesn't like "via"

```suggestion
You can configure MemGPT to use either a Qdrant server or an in-memory instance with the `memgpt configure` command.
```

File: qdrant-landing/content/documentation/frameworks/memgpt.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
```
Comment (mjang) [inline qdrant-landing/content/documentation/frameworks/memgpt.md:18]: General comment: While I used ` pip install -U pymemgpt` to set up the `memgpt` command, 

`memgpt configure` set up different prompts, which didn't match what you have here:

```
mike@Qdrant-Jang ~ $ memgpt configure
? Select LLM inference provider: (Use arrow keys)
 » openai
   azure
   local
```

I'm _guessing_ users need some other `memgpt` command to get to the list of storage options shown in your [MemGPT PR](https://github.com/cpacker/MemGPT/pull/1023/files#diff-ebba04859ec9cde65c68649dcd42eddc80428ca2d0de26d241bf0946210e6a30R555).

**Review by mjang — CHANGES_REQUESTED** [review]
Not complete. My guess: the reader may need a different `memgpt` command to get to the menus suggested in this PR.

File: qdrant-landing/content/documentation/frameworks/memgpt.md (hunk 2 above)
Comment (Anush008) [reply qdrant-landing/content/documentation/frameworks/memgpt.md:18]: You can see the storage options after selecting through these options.

Comment (Anush008) [reply qdrant-landing/content/documentation/frameworks/memgpt.md:18]: Currently you should see these options.

https://memgpt.readme.io/docs/storage

File: qdrant-landing/content/documentation/frameworks/memgpt.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+#### Configuring the Qdrant server.
+
```
Comment (mjang) [inline qdrant-landing/content/documentation/frameworks/memgpt.md:20]:
```suggestion

When you run `memgpt configure`, go through the prompts as described in the [MemGPT configuration documentation](https://memgpt.readme.io/docs/config).
After you address several `memgpt` prompts, you come to the following `memgpt` prompts:

```

File: qdrant-landing/content/documentation/frameworks/memgpt.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- Create perpetual chatbots that learn about you and change their own personalities over time.
+- Create perpetual chatbots that can interface with large data stores.
+
+Qdrant is available as a storage backend in MemGPT for storing and semantically retrieving data.
+
+## Usage
+
+You can configure MemGPT to use either a Qdrant server or an in-memory instance via the `memgpt configure` command.
+
+#### Configuring the Qdrant server.
```
Comment (mjang) [inline qdrant-landing/content/documentation/frameworks/memgpt.md:19]:
```suggestion
#### Configuring the Qdrant server
```

**Review by mjang — APPROVED** [review]
Approving with suggested updates. Thanks for explaining more of the process, @Anush008 .

**Comment by Anush008** [comment]
https://github.com/cpacker/MemGPT/pull/1023

**Comment by Anush008** [comment]
https://github.com/cpacker/MemGPT/pull/1023

**Pull request merged**